src/styles/antd-overrides/collapse.scss

```diff
@@ -169,3 +169,16 @@
 .ant-collapse-header-text {
   width: 100%;
 }
+
+// Create flow collapse styling
+.create-collapse {
+  .ant-collapse-header {
+    padding-left: 0 !important;
+    border-bottom: solid 1px var(--stroke-tertiary);
+  }
+
+  .ant-collapse-content-box {
+    padding-left: 0 !important;
+    padding-right: 0 !important;
```

Comment:
**question**: any way we can avoid `!important`? Probably greater specificity will do it.

Reply:
It is probable - I'm a noob at this stuff so I just found I had to add `!important` to make it work. Let me know a better way 🙏